web/slashes/scrapers.py

```diff
@@ -0,0 +1,218 @@
+import os
+import re
+import bs4
+import html
+from typing import List
+from flask import current_app, request
+
+from . import commands
+from ..executor import executor
+from ..utils import ForgetfulDict
+from .. import http, slack, formatting, utils
+
+
+class WebScrapingCommandHandler(commands.CommandHandler):
+    def __init__(self, *, relink_timeout):
```

Comment:
`relink_timeout` isn't quite right... maybe `term_timeout`, as in "search term timeout"?

Reply:
Maybe `rescrape_timeout`, using the verb from the superclass like we did for `relink_timeout`?

Reply:
👍

Reply:
011efcd